src/components/Recipient.jsx

```diff
@@ -0,0 +1,19 @@
+import styles from '../styles/recipient'
+
+import React from 'react'
+import { translate } from 'cozy-ui/react/I18n'
+import classNames from 'classnames'
+
+export const Recipient = ({ t, name, url, status }) => {
+  return (
+    <div className={classNames(styles['pho-recipient'])}>
```

Comment:
Using `classNames` is not necessary here, as it's only single classes. It'd be more readable without.